docs/pages/versions/unversioned/sdk/media-library.mdx

```diff
@@ -4,7 +4,7 @@ description: A library that provides access to the device's media library.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-media-library'
 packageName: 'expo-media-library'
 iconUrl: '/static/images/packages/expo-media-library.png'
-platforms: ['android', 'ios']
+platforms: ['android', 'ios', 'tvos']
```

Comment:
Can you also follow up with PR updating supported platforms in RND?

See: https://reactnative.directory/?search=expo-media-library

Reply:
@Simek thanks for the reminder! See https://github.com/react-native-community/directory/pull/1602